src/AutoRest.CSharp/Common/Generation/Writers/RequestWriterHelpers.cs

```diff
@@ -25,8 +25,7 @@ public static void WriteRequestCreation(CodeWriter writer, RestClientMethod clie
             var parameters = clientMethod.Parameters;
 
             var methodName = CreateRequestMethodName(clientMethod.Name);
-            var returnType = lowLevel ? typeof(Azure.Core.Request) : typeof(HttpMessage);
-            writer.Append($"{methodAccessibility} {returnType} {methodName}(");
+            writer.Append($"{methodAccessibility} {typeof(HttpMessage)} {methodName}(");
             foreach (Parameter clientParameter in parameters)
             {
                 if (lowLevel)
```

Comment:
I know it's unrelated to the change but why do we have this condition?

Reply:
If you remove this you get:

```
-        private HttpMessage CreateGetDocumentStatusRequest(Guid id, Guid documentId, RequestOptions requestOptions = null)
+        private HttpMessage CreateGetDocumentStatusRequest(Guid id, Guid documentId, RequestOptions requestOptions)
```

Reply:
And is that bad?

Reply:
We need requestOptions to be left off on API calls.

Reply:
But this only affects the CreateRequest methods is seems.